charts/rancher-backup/templates/s3-secret.yaml

```diff
@@ -14,13 +14,13 @@ stringData:
   credentialSecretNamespace: {{ required "When providing a Secret containing S3 credentials, a valid .Values.credentialSecretNamespace must be provided" .credentialSecretNamespace }}
   {{- end }}
   {{- if .region }}
-  region: {{ .region }}
+  region: {{ .region | quote }}
   {{- end }}
-  bucketName: {{ required "A valid .Values.bucketName is required for configuring S3 compatible storage as the default backup storage location" .bucketName }}
+  bucketName: {{ required "A valid .Values.bucketName is required for configuring S3 compatible storage as the default backup storage location" .bucketName | quote }}
   {{- if .folder }}
-  folder: {{ .folder }} 
+  folder: {{ .folder | quote }}
```

Comment:
Can you explain why this is needed in the commit or PR description. What string broke it.

Reply:
the commit message is updated.